packages/fxa-payments-server/server/lib/server.js

```diff
@@ -26,10 +26,21 @@ module.exports = () => {
 
   // Each of these config values (e.g., 'servers.content') will be exposed as the given
   // variable to the client/browser (via fxa-content-server/config)
-  const CLIENT_CONFIG_MAP = {
-    contentUrl: 'servers.content',
-    oAuthUrl: 'servers.oauth',
-    profileUrl: 'servers.profile',
+  const CLIENT_CONFIG = {
+    servers: {
+      content: {
+        url: config.get('servers.content.url'),
+      },
+      oauth: {
+        url: config.get('servers.oauth.url'),
+      },
+      profile: {
+        url: config.get('servers.profile.url'),
+      },
+    },
+    stripe: {
+      apiKey: config.get('stripe.apiKey'),
+    },
```

Comment:
I could have even done: `servers: config.get('servers')` but that seemed a tad to implicit to me. But maybe it should be that, I don't know...

Reply:
I think this is fine. We could play around with it more, if someday we accumulate a dozen more settings or something.